lib/chef-dk/command/generator_commands/base.rb

```diff
@@ -85,7 +85,13 @@ def setup_context
         def have_git?
           path = ENV["PATH"] || ""
           paths = path.split(File::PATH_SEPARATOR)
-          paths.any? {|bin_path| File.exist?(File.join(bin_path, "git#{RbConfig::CONFIG['EXEEXT']}"))}
+          exts = [RbConfig::CONFIG['EXEEXT']]
+          exts.concat(ENV["PATHEXT"].split(";")) unless ENV["PATHEXT"].nil?
+          paths.any? do |bin_path|
+            exts.any? do |ext|
+              File.exist?(File.join(bin_path, "git#{ext}"))
```

Comment:
I think we need a `return` so we break the loop. Otherwise we will return the last file we check... 🤔 

Something like this:
```ruby
return true if File.exist?(File.join(bin_path, "git#{ext}"))
```
and of course at the end of the function (outside the loop) we need to add `false`.